server/test.py

```diff
@@ -21,68 +22,17 @@ def tearDown(self):
         db.session.remove()
         db.drop_all()
 
-    def get_normalized_stats(self, player_name):
-        player = Player.query.filter_by(name=player_name).first()
-
-        if player:
-            stats = Stats.query.filter_by(player_id=player.id).first()
-            return stats.to_dict()
-        else:
-            fake_game_id = 0
-            fake_player_id = 0
-            stats = Stats(fake_game_id, fake_player_id)
-            return stats.to_dict()
 
     def test_basic_point(self):
         with open('data/test/basic_point.json') as f:
             game_str = f.read()
 
         self.client.post('/upload', data=game_str, content_type='application/json')
 
-        game = Game.query.filter_by(week=1).first()
-        rosters = json.loads(game.home_roster) + json.loads(game.away_roster)
-        for player_name in rosters:
-            normalized_stats = self.get_normalized_stats(player_name)
-
-            if player_name == 'Owen Lumley':
-                assert normalized_stats["pulls"] == 1
-            else:
-                assert normalized_stats["pulls"] == 0
-
-            if player_name in ['Sina Dee', 'Jamie Wildgen', 'Sebastien Belanger']:
-                assert normalized_stats["completions"] == 1
-            else:
-                assert normalized_stats["completions"] == 0
-
-            if player_name in ['Jamie Wildgen', 'Sebastien Belanger', 'Michael Colantonio']:
-                assert normalized_stats["catches"] == 1
-            else:
-                assert normalized_stats["catches"] == 0
-
-            if player_name == 'Michael Colantonio':
-                assert normalized_stats["goals"] == 1
-            else:
-                assert normalized_stats["goals"] == 0
-
-            if player_name == 'Sebastien Belanger':
-                assert normalized_stats["assists"] == 1
-            else:
-                assert normalized_stats["assists"] == 0
```

Comment:
The manual assertions were super handy to verify the calculations made sense. Now that we trust our output we can essentially use these tests to track any changes that happen to the calculation